Entity/SyncJob.php

```diff
@@ -17,7 +17,7 @@
  * Entity for data synchronization job.
  *
  * @ORM\Entity
- * @ORM\Table("@sync_jobs_table")
+ * @ORM\Table("ongr_sync_jobs")
```

Comment:
this whole class seems to be unused, and after investigation, probably will be removed as stated in #128